elements/lisk-chain/src/schema.ts

```diff
@@ -59,30 +77,50 @@ export const signingBlockHeaderSchema = {
 		'timestamp',
 		'height',
 		'previousBlockID',
-		'stateRoot',
-		'transactionRoot',
 		'generatorAddress',
-		'assets',
+		'transactionRoot',
+		'assetsRoot',
+		'stateRoot',
+		'maxHeightPrevoted',
+		'maxHeightGenerated',
+		'validatorsHash',
+		'aggregateCommit',
 	],
 };
 
 export const blockHeaderSchema = {
 	...signingBlockHeaderSchema,
-	$id: '/block/header/2',
+	$id: '/block/header/3',
 	required: [...signingBlockHeaderSchema.required, 'signature'],
 	properties: {
 		...signingBlockHeaderSchema.properties,
-		signature: { dataType: 'bytes', fieldNumber: 9 },
+		signature: { dataType: 'bytes', fieldNumber: 13 },
 	},
 };
 
 export const blockHeaderSchemaWithId = {
 	...blockHeaderSchema,
-	$id: '/block/header/2',
+	$id: '/block/header/3',
 	required: [...blockHeaderSchema.required, 'id'],
 	properties: {
 		...blockHeaderSchema.properties,
-		id: { dataType: 'bytes', fieldNumber: 10 },
+		id: { dataType: 'bytes', fieldNumber: 14 },
+	},
+};
+
+export const blockAssetSchema = {
```

Comment:
I think this would be more appropriate if we follow the naming conventions in the LIP. 
The LIP defines new block property `assets`, and since we have `blockHeaderSchema` for the property `header`, hence `blockAssetsSchema` would be inline.
```suggestion
export const blockAssetsSchema = {
```

Reply:
asset is not part of the block header, and we need to decode asset one by one. Notice block schema has byte array for `assets`